app/components/forms/wizard/attendee-step.js

```diff
@@ -1,8 +1,9 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import FormMixin from 'open-event-frontend/mixins/form';
+import EventWizardMixin from 'open-event-frontend/mixins/event-wizard';
 
-export default Component.extend(FormMixin, {
+export default Component.extend(FormMixin,EventWizardMixin, {
```

Comment:
Codacy found an issue: [A space is required after ','.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6140894)